cmd/operator-sdk/alpha/scorecard/cmd.go

```diff
@@ -25,10 +25,16 @@ import (
 
 func NewCmd() *cobra.Command {
 	var (
-		config string
-		//bundle   string // TODO - to be implemented
-		selector string
-		//list     bool // TODO - to be implemented
+		config         string
+		output         string
```

Comment:
it shows not rebased with the master. Could you please rebase it?

Reply:
should be updated with master now.